### PR TITLE
Fix several typos and grammar mistakes

### PR DESCRIPTION
**Pull request by MishimaHaruna**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

This is the result of proofreading some of the scripts (or just playing the game and paying attention to the dialogues), as well as mass-finding some common typos.

**Affected Branches:** 

- master

### Known Issues and TODO List

- No known issues. The changes have been tested in production.

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1838)

File: npc/merchants/socket_enchant.txt [hunk 1]
```diff
@@ -142,7 +142,7 @@
 			mes "Armor, you said?";
 			mes "Sure, no problem.";
 			mes "Armor is divided into 4 different classes, depending on the work difficulty.";
-			mes "C Class is the easiest one, and S Class he hardest one.";
+			mes "C Class is the easiest one, and S Class the hardest one.";
```
Comment (dastgir) [inline npc/merchants/socket_enchant.txt]: Is the?

File: npc/quests/quests_gonryun.txt [hunk 2]
```diff
@@ -3760,8 +3759,8 @@ gonryun,237,226,3	script	Han Ran Jiao#gon	4_M_TWMIDMAN,{
 		mes "here! Where did you get it!?";
 		next;
 		mes "[Han Ran Jiao]";
-		mes "I dropped that knife from being";
-		mes "careless. It belongs to one of my";
+		mes "I carelessly dropped it by accident.";
```
Comment (dastgir) [inline npc/quests/quests_gonryun.txt]: Carelessly dropped by accident.
Something doesn't sound good here.
Maybe "I carelessly dropped the knife?"

**Comment by MishimaHaruna** [comment]
Thank you! Amended the commit

**Comment by MishimaHaruna** [comment]
I'm merging this - if further changes are necessary, we can make follow-up commits later.

**Pull request merged**